src/database/migrations/20190908184324-create-travel-request.js

```diff
@@ -0,0 +1,93 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('travel_requests', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      user_id: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: {
+            tableName: 'users',
+          },
+          key: 'id'
+        }
+      },
+      travel_type: {
+        type: Sequelize.ENUM,
+        allowNull: false,
+        values: ['one-way', 'return']
+      },
+      origin: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+        validate: {
+          msg: 'Please provide the origin of travel'
+        }
+      },
+      destination: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+        validate: {
+          msg: 'Please provide your destination'
+        }
+      },
+      departure_date: {
+        type: Sequelize.DATE,
+        allowNull: false,
+        validate: {
+          msg: 'Please provide the date of your departure'
+        }
+      },
+      return_date: {
+        type: Sequelize.DATE,
+        allowNull: true,
+        validate: {
+          msg: 'Please provide the date of your return'
+        }
+      },
+      travel_purpose: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+        validate: {
+          msg: 'Please provide reason for travel'
+        }
+      },
+      accommodation_id: {
```

Comment:
I will suggest we use string datatype for our accommodation ID so that it can accommodate multi-city travel requests in which case we can separate the IDs with commas.

Reply:
@chuksjoe I am not really getting the point. please could you be more explicit, maybe come up with a scenario?